packages/stryker-webpack-transpiler/test/helpers/sinonInit.ts

```diff
@@ -1,5 +1,5 @@
 import * as sinon from 'sinon';
-import * as log4js from 'log4js';
+import * as log4js from 'stryker-api/logging';
```

Comment:
we call it log4js here and import the entire module

Reply:
Refactored the name, but the entire module is needed in order to mock away the `getLogger` method.